draft-ietf-tsvwg-careful-resume.xml

```diff
@@ -1011,13 +1011,43 @@ Examples of the transitions between phases are provided in <xref target="Example
 		<figure anchor="qlog-def">
 	         <!--- 	<name>cr_phase</name> -->
 <sourcecode type="cddl"><![CDATA[
-XX Fix me as CDDL XX
-<!-- CrState::OBSERVE => { 0 }
-CrState::RECON => { 1 }
-CrState::UNVAL => { 2 }
-CrState::VALIDATE => { 3 }
-CrState::NORMAL => { 4 }
-CrState::RETREAT => { 100 } -->
+RecoveryCarefulResumePhaseUpdated = {
+? old: CarefulResumePhase,
+new: CarefulResumePhase,
+
+
+state_data: CarefulResumeStateParameters,
+? restored_data: CarefulResumeRestoredParameters,
+
+
+? trigger: "packet_loss"	;Loss triggers moving to safe retreat
+"cwnd_limited"			;trigger for moving to unvalidated
+"cr_mark_acknowledged"		;trigger for moving to validating or normal
+"rtt_not_validated"		;trigger for moving to normal, when CR not allowed
+"ECN_CE"			;trigger for moving to safe retreat.
+"exit_recovery"			;trigger for moving to normal, 1 rtt after a congestion event 
+}
+
+
+CarefulResumePhase = " reconnaissance " 
```

Comment:
Do you really need literal spaces in the value? I would avoid whitespace whereever possible.